.teamcity/src/subprojects/build/core/CoreBuild.kt

```diff
@@ -9,10 +9,8 @@ import subprojects.build.*
 class CoreBuild(private val osJVMComboEntry: OSJDKEntry) : BuildType({
     id("KtorMatrix_${osJVMComboEntry.osEntry.name}${osJVMComboEntry.jdkEntry.name}".toExtId())
     name = "${osJVMComboEntry.jdkEntry.name} on ${osJVMComboEntry.osEntry.name}"
-    artifactRules = """
-                        +:**/build/**/*.jar
-                        $reportArtifacts
-                    """.trimIndent()
+    artifactRules = """+:**/build/**/*.jar
```

Comment:
Could you indent this and the following changed blocks properly for readability?

Reply:
I unindented it on purpose because I'm getting some horrible formatting in the output on TeamCity, and it seems it may be related. If it's for readability of code, then ignore this pull request.